nailed-it/New Group/TryItOnViewController.swift

```diff
@@ -20,10 +19,42 @@ class TryItOnViewController: UIViewController, UIImagePickerControllerDelegate,
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        selectColorButton.isEnabled = false
         
+        // Load instance of Polish Library
         let storyboard = UIStoryboard(name: "Main", bundle: nil)
         polishLibraryViewController = storyboard.instantiateViewController(withIdentifier: "PolishLibraryViewController") as! PolishLibraryViewController
+        
+        // Get initial image mask data if it exists
+        if let loadedImageData = UserDefaults.standard.object(forKey: "savedImage") as? Data {
+            if let loadedMaskData = UserDefaults.standard.object(forKey: "savedMask") as? [String:Any] {
+                image = UIImage(data: loadedImageData)!
+                mask = TCMask(data: loadedMaskData["data"] as! [UInt8], size: CGSizeFromString(loadedMaskData["size"] as! String))
+                
+                applyImageMask()
+            }
+        } else {
+            selectColorButton.isEnabled = false
+        }
+    }
+    
+    func applyImageMask() {
+        // adjust the size of image view to make it fit the image size and put it in the center of screen
+        var x:CGFloat, y:CGFloat, width:CGFloat, height:CGFloat
```

Comment:
why are we adding this stuff back in?

Reply:
oh whoops, I think my rebase messed up and I thought it was still around.

Reply:
https://github.com/tolzar/nailed-it/pull/18/files#diff-f17fb3c95c0e84f1802724b850704306L59